scripts/minikube.sh

```diff
@@ -118,6 +121,16 @@ deploy-rook)
     DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
     "$DIR"/rook.sh deploy
     ;;
+create-block-pool)
```

Comment:
This is a glue between `rook.sh` and `minikube.sh`, just so that the travis scripts only invoke `minikube.sh`. I would assume their presence in rook.sh helps running them on non-minikube environments.